pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -313,7 +313,9 @@ func (n *NodeLabeller) removeLabellerLabels(node *v1.Node) {
 			strings.Contains(label, kubevirtv1.CPUTimerLabel) ||
 			strings.Contains(label, kubevirtv1.HypervLabel) ||
 			strings.Contains(label, kubevirtv1.RealtimeLabel) ||
-			strings.Contains(label, kubevirtv1.SEVLabel) {
```

Comment:
I would probably make separate PR for this one.

Reply:
There you go: https://github.com/kubevirt/kubevirt/pull/7928
I've assigned you to it, please take a look